beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -855,6 +858,96 @@ func (bs *Server) GetBlockRoot(w http.ResponseWriter, r *http.Request) {
 	http2.WriteJson(w, response)
 }
 
+// ListCommittees retrieves the committees for the given state at the given epoch.
+// If the requested slot and index are defined, only those committees are returned.
+func (bs *Server) ListCommittees(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.ListCommittees")
+	defer span.End()
+
+	stateId := mux.Vars(r)["state_id"]
+	if stateId == "" {
+		http2.HandleError(w, "state_id is required in URL params", http.StatusBadRequest)
+		return
+	}
+
+	ok, _, e := shared.UintFromQuery(w, r, "epoch")
+	if !ok {
+		return
+	}
+	ok, _, i := shared.UintFromQuery(w, r, "index")
+	if !ok {
+		return
+	}
+	ok, _, s := shared.UintFromQuery(w, r, "slot")
+	if !ok {
+		return
+	}
+
+	st, err := bs.Stater.State(ctx, []byte(stateId))
+	if err != nil {
+		helpers.PrepareStateFetchHTTPError(w, err)
+		return
+	}
+
+	epoch := slots.ToEpoch(st.Slot())
+	if e != 0 {
+		epoch = primitives.Epoch(e)
+	}
+	activeCount, err := corehelpers.ActiveValidatorCount(ctx, st, epoch)
+	if err != nil {
+		http2.HandleError(w, "Could not get active validator count: "+err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	startSlot, err := slots.EpochStart(epoch)
+	if err != nil {
+		http2.HandleError(w, "Invalid epoch: "+err.Error(), http.StatusBadRequest)
```

Comment:
```suggestion
		http2.HandleError(w, "Could not get epoch start slot: "+err.Error(), http.StatusInternalServerError)
```
This should not be a bad request, but a server error.